### PR TITLE
RichCommands: Navigate Up, Back and Forward

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #11481 

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [x] Did you remove any strings from the en-us resource file?
   - [x] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [x] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Navigate to folders
   2. Use toolbar buttons and shortcuts to navigate up, back and forward

### CODE REVIEW

File: src/Files.App/UserControls/AddressToolbar.xaml [hunk 1]
```diff
@@ -65,48 +65,36 @@
...
 				</Button>
 
 				<Button
 					x:Name="Up"
 					x:FieldModifier="public"
 					AccessKey="U"
-					AutomationProperties.Name="{helpers:ResourceString Name=NavUpButton/AutomationProperties/Name}"
-					Command="{x:Bind ViewModel.UpClickCommand, Mode=OneWay}"
-					IsEnabled="{x:Bind ViewModel.CanNavigateToParent, Mode=OneWay}"
+					AutomationProperties.Name="{x:Bind ViewModel.Commands.NavigateUp.Label, Mode=OneWay}"
```
[the remark this answers is not in the document]
Comment (ferrariofilippo) [reply src/Files.App/UserControls/AddressToolbar.xaml:92]: Yes

[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/UserControls/AddressToolbar.xaml:92]: What was it before?

[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/UserControls/AddressToolbar.xaml:92]: Can you add `AutomationProperties.FullDescription` as well and use the description property? I feel that the `Name` property doesn't provide enough context on it's own.

File: src/Files.App/UserControls/AddressToolbar.xaml [hunk 2]
```diff
@@ -65,48 +65,39 @@
...
+					IsEnabled="{x:Bind ViewModel.Commands.NavigateBack.IsExecutable, Mode=OneWay}"
 					Style="{StaticResource AddressToolbarButtonStyle}"
-					ToolTipService.ToolTip="{helpers:ResourceString Name=NavBackButton/ToolTipService/ToolTip}">
-					<FontIcon FontSize="14" Glyph="&#xE72B;" />
-					<Button.KeyboardAccelerators>
-						<KeyboardAccelerator Key="XButton1" />
-						<KeyboardAccelerator Key="Back" />
-						<KeyboardAccelerator Key="Left" Modifiers="Menu" />
-					</Button.KeyboardAccelerators>
+					ToolTipService.ToolTip="{x:Bind ViewModel.Commands.NavigateBack.Description, Mode=OneWay}">
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml:73]:
```suggestion
					ToolTipService.ToolTip="{x:Bind ViewModel.Commands.NavigateBack.LabelWithHotKey, Mode=OneWay}">
```

File: src/Files.App/UserControls/AddressToolbar.xaml [hunk 3]
```diff
@@ -65,48 +65,39 @@
...
+					AutomationProperties.Name="{x:Bind ViewModel.Commands.NavigateUp.Label, Mode=OneWay}"
+					Command="{x:Bind ViewModel.Commands.NavigateUp, Mode=OneWay}"
+					IsEnabled="{x:Bind ViewModel.Commands.NavigateUp.IsExecutable, Mode=OneWay}"
 					Style="{StaticResource AddressToolbarButtonStyle}"
-					ToolTipService.ToolTip="{helpers:ResourceString Name=NavUpButton/ToolTipService/ToolTip}">
-					<FontIcon FontSize="14" Glyph="&#xE74A;" />
-					<Button.KeyboardAccelerators>
-						<KeyboardAccelerator Key="Up" Modifiers="Menu" />
-					</Button.KeyboardAccelerators>
+					ToolTipService.ToolTip="{x:Bind ViewModel.Commands.NavigateUp.Description, Mode=OneWay}">
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml:99]:
```suggestion
					ToolTipService.ToolTip="{x:Bind ViewModel.Commands.NavigateUp.LabelWithHotKey, Mode=OneWay}">
```

File: src/Files.App/UserControls/AddressToolbar.xaml [hunk 4]
```diff
@@ -65,48 +65,39 @@
...
+					Command="{x:Bind ViewModel.Commands.NavigateForward, Mode=OneWay}"
+					IsEnabled="{x:Bind ViewModel.Commands.NavigateForward.IsExecutable, Mode=OneWay}"
 					Style="{StaticResource AddressToolbarButtonStyle}"
-					ToolTipService.ToolTip="{helpers:ResourceString Name=NavForwardButton/ToolTipService/ToolTip}">
-					<FontIcon FontSize="14" Glyph="&#xE72A;" />
-					<Button.KeyboardAccelerators>
-						<KeyboardAccelerator Key="XButton2" />
-						<KeyboardAccelerator Key="Right" Modifiers="Menu" />
-					</Button.KeyboardAccelerators>
+					ToolTipService.ToolTip="{x:Bind ViewModel.Commands.NavigateForward.Description, Mode=OneWay}">
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml:86]:
```suggestion
					ToolTipService.ToolTip="{x:Bind ViewModel.Commands.NavigateForward.Description, Mode=OneWay}">
```

**Review by yaira2 — DISMISSED** [review]
LGTM

**Comment by yaira2** [comment]
@ferrariofilippo can you fix the merge conflict?

**Pull request merged**